src/main/java/owlmoney/model/bank/Saving.java

```diff
@@ -61,6 +61,29 @@ public Saving(String name, double currentAmount, double income) {
         this.storage = new Storage(FILE_PATH);
     }
 
+    /**
+     * Creates an instance of a savings account.
+     *
+     * @param name          The name of the bank account.
+     * @param currentAmount The current amount of money in  the bank account.
+     * @param income        The amount of money that is credited monthly into the account.
+     */
+    public Saving(String name, double currentAmount, double income, Date nextIncomeDate) {
+        super(name, currentAmount);
+        this.income = income;
+        this.type = SAVING;
+        this.transactions = new TransactionList();
+        this.recurringExpenditures = new RecurringExpenditureList();
+        Calendar calendar = Calendar.getInstance();
```

Comment:
If these calendar codes are not used in this overloaded function, we should remove it